ssl/statem/statem_srvr.c

```diff
@@ -3130,6 +3130,8 @@ static int tls_process_cke_dhe(SSL *s, PACKET *pkt)
         goto err;
     }
     cdh = EVP_PKEY_get0_DH(ckey);
+    if (cdh == NULL)
+        goto err;
```

Comment:
A call to SSLFatal is missing here , see `EVP_PKEY_get0_DH` call from `statem_clnt.c`

Reply:
I've updated the code, should I create a new PR?

Reply:
No, just do 'git push -f' to push it to your branch and this PR will be updated.